PowerEditor/src/WinControls/shortcut/shortcut.cpp

```diff
@@ -483,7 +482,7 @@ INT_PTR CALLBACK Shortcut::run_dlgProc(UINT Message, WPARAM wParam, LPARAM)
 // return true if one of CommandShortcuts is deleted. Otherwise false.
 void Accelerator::updateShortcuts() 
 {
-	vector<int> incrFindAccIds;
+	vector<UINT> incrFindAccIds;

```

Comment:
Oh indeed. Thanks. I corrected my previous comment.  But still I think it's safe and better to use minimal required unsigned type here. Why to increase memory usage impact? )